doc/installing.rst

```diff
@@ -130,13 +129,63 @@ Install from the git source
 
 If ``pip`` is unsuccessful or if you are planning to use any of the
 ``PypeIt`` development branches, then you should install directly
-from GitHub.
+from GitHub. These instructions assume that you are using an anaconda python
+installation.
 
- #. Clone the repository::
+#. Note it would be a very good idea to nuke your current anaconda installation and start
```

Comment:
Nuking one's conda installation is unnecessary. After cloning the pypeit repo, users only need to execute the commands `conda env create -f environment.yml && conda activate pypeit` to have all dependencies installed and the environment prepared for a developer install `pip install -e .`/`python setup.py develop`. If check_requirements complains about not finding PySide2, I would honestly recommend just commenting it out of `requirements.txt`.

Reply:
@finagle29 you can go ahead and edit these files as you see fit or punt until your next PR.